src/quickfix.c

```diff
@@ -4991,6 +5018,11 @@ qf_jump_first(qf_info_T *qi, int_u save_qfid, int forceit)
     if (qf_restore_list(qi, save_qfid) == FAIL)
 	return;
 
+
+    if (!check_can_set_curbuf_forceit(forceit))
+	return;
```

Comment:
and here

Reply:
Should be used already in 

Test_cbuffer()
Test_cexpr()
Test_cfile()
Test_grep()

Reply:
okay